### PR TITLE
Fr 26 create user helper function that updates ideas

### PR DESCRIPTION
**Pull request by VladimirKarin**
https://vladimirkarin.atlassian.net/browse/FR-26

### CODE REVIEW

File: models/ideas.js [hunk 1]
```diff
@@ -14,6 +15,18 @@ function createIdea(header, description, askedSum, userId) {
     return newUser;
 }
 
+function updateIdea(ideaId, idea) {
+    let ideas = getIdeas();
+    const idea = idea;
+
+    const updateIdeas = ideas.reduse((updateIdeas, idea) => {
+        if (idea.id === ideaId) {
+            return [updateIdeas, updateIdea]
```
Comment (fedosejev) [inline models/ideas.js:24]: There's an issue with this line - can you spot it?

File: models/ideas.js [hunk 2]
```diff
@@ -23,6 +36,7 @@ function deleteIdea(ideaId) {
 }
 
 module.exports = {
-    deleteIdea,
-    createIdea
+    createIdea,
+    updateIdea,
+    deleteIdea
```
Comment (fedosejev) [inline models/ideas.js:41]: Can you format this file with Prettier?

File: models/ideas.js (hunk 1 above)
Comment (VladimirKarin) [reply models/ideas.js:24]: updateIdeas not coppied? [...updateIdeas, updateIdea] ?

File: models/ideas.js (hunk 2 above)
Comment (VladimirKarin) [reply models/ideas.js]: done

File: models/ideas.js (hunk 1 above)
Comment (fedosejev) [reply models/ideas.js]: Yes

File: models/ideas.js [hunk 3]
```diff
@@ -14,6 +15,19 @@ function createIdea(header, description, askedSum, userId) {
     return newUser;
 }
 
+function updateIdea(ideaId, idea) {
+    let ideas = getIdeas();
+    const idea = idea;
+
+    const updateIdeas = ideas.reduse((updateIdeas, idea) => {
+        if (idea.id === ideaId) {
+            return [...updateIdeas, updateIdea];
```
Comment (fedosejev) [inline models/ideas.js:24]: What is `updateIdea` here?

File: models/ideas.js [hunk 4]
```diff
@@ -14,6 +15,19 @@ function createIdea(header, description, askedSum, userId) {
     return newUser;
 }
 
+function updateIdea(ideaId, idea) {
+    let ideas = getIdeas();
+    const idea = idea;
```
Comment (fedosejev) [inline models/ideas.js:20]: Why do we need this?

File: models/ideas.js [hunk 5]
```diff
@@ -14,6 +15,19 @@ function createIdea(header, description, askedSum, userId) {
...
+
+    const updateIdeas = ideas.reduse((updateIdeas, idea) => {
```
Comment (fedosejev) [inline models/ideas.js:22]: Typo

File: models/ideas.js (hunk 3 above)
Comment (VladimirKarin) [reply models/ideas.js:24]: it should be idea that we want to update. and it comes in params.

File: models/ideas.js (hunk 4 above)
Comment (VladimirKarin) [reply models/ideas.js:20]: We don't, trying to wright down it another way

**Pull request closed without merging**